requirements.txt

```diff
@@ -1,3 +1,6 @@
 fastapi==0.63.0
+gunicorn==20.1.0
+httptools==0.2.0
```

Comment:
Are httptools and uvloop dependencies of gunicorn? If so, let's let the requirements-lock handle the installation of those.

Reply:
They actually aren't, according to pip. I'm not sure why not, because we get errors without them. Maybe they are not required under certain configurations?